aten/src/ATen/native/mkldnn/xpu/detail/QMatmul.cpp

```diff
@@ -0,0 +1,272 @@
+#include <ATen/Tensor.h>
+#include <ATen/core/Tensor.h>
+
+#include <ATen/native/mkldnn/xpu/detail/oneDNNContext.h>
+#include <ATen/native/mkldnn/xpu/detail/Attr.h>
+
+#include <oneapi/dnnl/dnnl.hpp>
+
+namespace at::native::onednn {
+
+void quantized_matmul(
```

Comment:
Suggest to add more descriptions about this kernel, such as the functionality: which kinds of quantization strategy supported in this kernel, asymmetric, per-tensor for activation, asymmetric, per-tensor for weight?

Reply:
Thanks for your suggestions. I have added more detailed description in a comment at codes belowing.